packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_TotalElementLoop_DP.cpp

```diff
@@ -58,20 +58,18 @@
 
 using namespace TpetraExamples;
 
+#define COMM_T Teuchos::RCP<const Teuchos::Comm<int> >
```

Comment:
See above.  Make this a typedef, not a macro.  Also, how much effort does this relabeling really save?